File: Núcleo/src/LogicaNegocio/IMedico.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package LogicaNegocio;

import java.util.Date;

/**
 *
 * @author CrizUP
 */
public interface IMedico {
    public boolean registrar(Medico coordinador);
    public boolean modificar(Medico coordinador);
    public boolean registrarEntrada(String numeroPersonal, String numeroConsultorio);
    public boolean registrarSalida(String numeroPersonal);
    public boolean eliminar();
    public Medico obtenerMedico(String numeroPersonal);    
}

File: Nucleo servidor web/application/libraries/IMedico.php
<?php
interface IMedico{
    public function registrar($medico,$idUsuario);
    public function modificar($medico);
    public function registrarEntrada($rfc, $numeroConsultorio);
    public function registrarSalida($rfc);
    public function eliminar($rfc);
    public function obtenerMedico($rfc);  
    public function obtenerMedicoId($idUsuario);
    public function obtenerMedicos();
}